mithril.js

```diff
@@ -1507,16 +1465,40 @@
 			controllers[index].onunload(event)
 		}
 
-		return checkPrevented(component, root, index, isPrevented)
-	}
+		if (!isPrevented) {
```

Comment:
Go ahead and tack a `// eslint-disable-line max-statements` to the function if necessary.